acceptance-tests/src/test/java/me/atam/atam4jsampleapp/testsupport/Atam4jApplicationStarter.java

```diff
@@ -9,11 +9,12 @@
 public class Atam4jApplicationStarter {
 
     public static DropwizardTestSupport<ApplicationConfiguration> startApplicationWith(int initialDelayInMillis,
-                                                                                       Class testClass) {
+                                                                                       Class testClass, int periodInMillis) {
```

Comment:
Please break long lines (lines spanning the default 120 column width in IntelliJ - https://www.jetbrains.com/help/idea/2016.1/code-style-java.html?origin=old_help#d58663e279) for improved code readability.